Tests/scripts/hook_validations/secrets.py

```diff
@@ -212,9 +209,9 @@ def create_temp_white_list(file_contents):
     return temp_white_list
 
 
-def retrieve_related_yml(file_path_temp):
+def retrieve_related_yml(file_dir_path_temp):
     matching_yml_file_contents = None
-    yml_file = file_path_temp + '.yml'
+    yml_file = file_dir_path_temp + '/' + file_dir_path_temp.split('/')[-1] + '.yml'
```

Comment:
```suggestion
    yml_file = os.path.join(file_dir_path_temp, os.path.basename(file_dir_path_temp) + '.yml')
```

Reply:
I just read why it's important to use the os.path module (Thanks)